cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -514,7 +514,7 @@
         /etc/ld.so.cache r,
         /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}ld{-*,64}.so* mrix,
         # libc, you are funny
-        /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}libc{,-[0-9]*}.so* mr,
+        /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,glibc-hwcaps/{x86-64-v{2,3,4},power{9,10},z1{3,4,5,6}}/,@{multiarch}/{,atomics/}}libc{,-[0-9]*}.so* mr,
```

Comment:
sorry for being picky, here too for consistency

Reply:
> sorry for being picky, here too for consistency

Oops, my mistake, fixed now.

Reply:
thanks